resume.html

```diff
@@ -0,0 +1,171 @@
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>wommy.io - resume</title>
+
+		<link rel='stylesheet' href='style.css'>
+		<style>
+			#main {
+				margin: 1rem;
+			}
+			h2 {
+				border-bottom: 1px solid silver;
+				padding-bottom: .3rem;
+			}
+			.margin{
+				margin: 1rem 0;
+			}
+			.indent{
+				padding-left: 1.25rem;
+			}
+			.highlight{
+				background-color: gainsboro;
+				border-radius: 3px;
+				padding: .2em .4em;
+				font-size: 85%;
+				font-family: monospace;
+			}
+		</style>
+	</head>
+	<body id='resume'>
+		<nav>
+			<a href='./index.html'>home</a>
+		</nav>
+		<div id='main'>
+			<h1 style="display:none;">resume</h1>
+			<div id='content'>
+				<h2 style="display:none;">contact</h2>
+					<div class='margin'>
+						<img src="./wommy.jpeg" alt="headshot" style="float:left; margin-right:20px;width:40%;">
+						<div><a href="./resume.pdf">download pdf</a></div>
+						<h2>TOMMY WILLIAMS</h2>
+						<div><span class='highlight'>email:</span> <a href='mailto:tom@technomad.media?subject=resume' target="_blank">tom@technomad.media</a></div>
+						<div><span class='highlight'>phone:</span> <a href='tel:3128189011' target="_blank">312 818 9011</a></div>
+						<div><span class='highlight'>portfolio:</span> <a href="github.com/wommy" target="_blank">github.com/wommy</a>, <a href="wommy.io" target="_blank">wommy.io</a>, <a href="technomad.media" target="_blank">technomad.media</a></div>
+					</div>
+					<div class='margin'>
+						<div><span class='highlight'>focus:</span> digital publishing, media distribution, ecommerce</div>
+						<div><span class='highlight'>location:</span> chicago, il; royal oak, mi; newyork, ny</div>
+						<div><span class='highlight'>hobbies:</span> cycling, martial arts, dancing, techno</div>
+						<div><span class='highlight'>built with:</span> linux, github, vscode, html, css</div>
+						<div><span class='highlight'>view online:</span> github.com/wommy/resume</div>
+					</div>
+				<h2>education</h2>
+					<div>class of 2008: university of detroit jesuit high school</div>
+					<div>2008 - 2010: columbia college chicago</div>
+					<div>class of 2015: depaul university</div>
+					  <div class='indent'>bachelors of art - psychology</div>
+					  <div class='indent'>minor - interactive and social media</div>
+				<h2>experience</h2>
+					<div class='margin'>
+						<div>since 2010: technomad:</div>
+						<div>web freelancer: 24+ microbusinesses; development, design, consultation</div>
+					</div>
+					<div class='margin'>
+						<div>2015 - 2017: solution people:</div>
+						<div>lead design, development, content, database; 30+ websites</div>
+					</div>
+					<div class='margin'>
+						<div>2015 summer, fall: original graphics printing:</div>
+						<div>subcontractor; 5+ brands: website, signage, decals, tshirts</div>
+					</div>
+				<h2>conferences / meetups</h2>
```

Comment:
Are these events you have attended, or spoken at? People do not typically list events they have attended on their resume

Reply:
i have spent a great deal of time and money traveling to these 

i am proud of them as they illustrate who i am

Reply:
I would make sure to mention this explicitly then.